src/main/resources/data-config-empty.json

```diff
@@ -0,0 +1,41 @@
+{
+    "project-settings": {
+        "year": "current",
```

Comment:
reminder to update this to `copyright-year`

Reply:
Same, will update this over.